ClinGen/clincoded
# PR for issue #2075: Update HPO Browser links

Updates all occurrence of previous HPO link base to new HPO link base:

```
http://compbio.charite.de/hpoweb/showterm?id=<HPO id>
=>
https://hpo.jax.org/app/browse/term/<HPO id>
```

Based on the 9 locations in issue #2075  the test steps are as follows:
1. Group curation form page: https://2075-sc-update-hpo-links.demo.clinicalgenome.org/group-curation/. Click on the "HPO" link to verify it links to the correct external web page.
1. Family curation form page: https://2075-sc-update-hpo-links.demo.clinicalgenome.org/family-curation/. Click on the "HPO" link.
1. Individual curation form page: https://2075-sc-update-hpo-links.demo.clinicalgenome.org/individual-curation/. Click on the "HPO" link.
1. Case control evidence form: https://2075-sc-update-hpo-links.demo.clinicalgenome.org/case-control-curation/. Click on the "HPO" link.


Experimental curation test steps: navigate to this url as a base link to carry out the series of test items below https://2075-sc-update-hpo-links.demo.clinicalgenome.org/experimental-curation/

1. . Biochemical function: navigate to base link and select as the image below. Upon selection, you should see the biochemical function form appear. Click on the "HPO" link.

![Screen Shot 2019-11-26 at 3 33 51 PM](https://user-images.githubusercontent.com/57968339/69681269-6bd23880-1062-11ea-86b2-502cfb2d841e.png)

2. Model systems: navigate to base link and select as the image below. The Model system form should show up at the bottom. Click on the "HPO" link.
![Screen Shot 2019-11-26 at 3 52 33 PM](https://user-images.githubusercontent.com/57968339/69681986-c9678480-1064-11ea-9409-be77eee6d0b7.png)

3. Rescue: navigate to base link and select as the image below. The Rescue form should show up at the bottom. Click on the "HPO" link.

![Screen Shot 2019-11-26 at 3 53 45 PM](https://user-images.githubusercontent.com/57968339/69682032-f3b94200-1064-11ea-96a7-729ca8872e46.png)

4. Creating new GDM - Add Disease: as an example, go to family curation form page: https://2075-sc-update-hpo-links.demo.clinicalgenome.org/family-curation/
    1. In section "Family – Disease(s) & Phenotype(s)", press button "Disease +".
    1. Check the check box at the lower bottom of the poped-up modal. This will change the content of the modal.
    1. The link "HPO-Browser" should appear. Click on the link to verify it links to the correct external web page.

Test items and steps in VCI:
1. Adding/Changing disease: go to home page, and on left navigation panel click "View list of all Variant Interpretations".
1. Click on any of the variant id to access detail page. See if there's a "Interpretation +" button as below image. If not, change a variant id and access its detail page again till you see the button. Click on the button to open the modal.

![Screen Shot 2019-11-26 at 4 08 46 PM](https://user-images.githubusercontent.com/57968339/69682675-0b91c580-1067-11ea-9ebc-23142b1baba7.png)

3. Agree the term. It will bring you to the page as below image:

![Screen Shot 2019-11-26 at 4 10 20 PM](https://user-images.githubusercontent.com/57968339/69682734-43990880-1067-11ea-910a-867d402db62a.png)

4. Click on the button "Disease +" and a modal will pop up.
5. Check the checkbox near the bottom of the modal.
6. The link "HPO-Browser" should appear. Click on the link to verify it links to the correct external web page.
